explorer/templates.go

```diff
@@ -251,8 +251,9 @@ func makeTemplateFuncMap(params *chaincfg.Params) template.FuncMap {
 			result = int(a) * b
 			return
 		},
-		"TimeConversion": func(a uint64) (result time.Time) {
-			result = time.Unix(int64(a), 0)
+		"TimeConversion": func(a uint64) (result string) {
+			dateTime := time.Unix(int64(a), 0)
+			result = dateTime.Format("2006-01-02 15:04:05 MST")
```

Comment:
Note that this will only say "UTC" if the host machines clock is on UTC.  To force UTC display (desirable, especially for public machines):

    dateTime := time.Unix(int64(155341541), 0).UTC()